src/zope/session/http.py

```diff
@@ -130,7 +130,7 @@ class ICookieClientIdManager(IClientIdManager):
         )
 
 
-@implementer(IClientIdManager, ICookieClientIdManager)
+@implementer(ICookieClientIdManager)
```

Comment:
`class ICookieClientIdManager(IClientIdManager):` is right above this, so this LGTM.